ivy/data_classes/container/experimental/manipulation.py

```diff
@@ -4065,6 +4065,180 @@ def trim_zeros(
+        applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        x
+            input array
+        shape
+            array indices. Must have an integer data type.
+        dim
+            axis over which to select values. If `axis` is negative,
+            the function must determine the axis along which to select values
+            by counting from the last dimension.
+            By default, the flattened input array is used.
+        out
+            optional output array, for writing the result to. It must
+            have a shape that the inputs broadcast to.
+        key_chains
+            The key-chains to apply or not apply the method to.
+            Default is ``None``.
+        to_apply
+            If True, the method will be applied to key_chains,
+            otherwise key_chains will be skipped. Default is ``True``.
+        prune_unapplied
+            Whether to prune key_chains for which the function was
+            not applied. Default is ``False``.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples).
+            Default is ``False``.
+
+        Returns
+        -------
+            ret
+                an array having the same data type as `x`.
+                The output array must have the same rank
+                (i.e., number of dimensions) as `x` and
+                must have the same shape as `x`,
+                except for the axis specified by `axis`
+                whose size must equal the number of elements in `indices`.
+
+
+        Examples
+        --------
+        With 'ivy.Container' input:
+
+        >>> x = ivy.Container(a = ivy.array([[True, False, False, True],
+                                            [False, True, False, True]])),
+        ...                     b = ivy.array([[1.2, 2.3, 3.4, 4.5],
+                                            [5.6, 6.7, 7.8, 8.9]]),
+        ...                     c = ivy.array([[1, 2, 3, 4],
+                                            [5, 6, 7, 8]]))
+        >>> dim = 1
+        >>> shape = (2, 2)
+        >>> y = ivy.Container._static_unflatten(x, shape=shape, dim=dim)
+        >>> print(y)
+        {
+            a: ivy.array([[[True, False], [False, True]],
+                        [[False, True], [False, True]]])
+            b: ivy.array([[[1.2, 2.3], [3.4, 4.5]], [[5.6, 6.7], [7.8, 8.9]]])
+            c: ivy.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]])
+        }
+        """
+        return ContainerBase.cont_multi_map_in_function(
+            "unflatten",
+            x,
+            shape=shape,
+            dim=dim,
+            out=out,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+        )
+
+    def unflatten(
+        self: Union[int, ivy.Array, ivy.NativeArray, ivy.Container],
```

Comment:
Same error as in the array class, "self" can only be "ivy.container".

Reply:
Corrected 👍